plugins/webp-uploads/hooks.php

```diff
@@ -534,22 +534,26 @@ function webp_uploads_update_image_references( $content ): string {
 
 	// This content does not have any tag on it, move forward.
 	// TODO: Eventually this should use the HTML API to parse out the image tags and then update them.
-	if ( ! preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
+	if ( 0 === (int) preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
 		return $content;
 	}
 
 	$images = array();
 	foreach ( $img_tags as list( $img ) ) {
 		$processor = new WP_HTML_Tag_Processor( $img );
-		if ( ! $processor->next_tag( array( 'tag_name' => 'IMG' ) ) ) {
+		if ( 0 === (int) $processor->next_tag( array( 'tag_name' => 'IMG' ) ) ) {
 			// This condition won't ever be met since we're iterating over the IMG tags extracted with preg_match_all() above.
 			continue;
 		}
 
 		// Find the ID of each image by the class.
 		// TODO: It would be preferable to use the $processor->class_list() method but there seems to be some typing issues with PHPStan.
 		$class_name = $processor->get_attribute( 'class' );
-		if ( ! is_string( $class_name ) || ! preg_match( '/(?:^|\s)wp-image-([1-9]\d*)(?:\s|$)/i', $class_name, $matches ) ) {
+		if (
+			! is_string( $class_name )
+			||
+			1 !== preg_match( '/(?:^|\s)wp-image-([1-9]\d*)(?:\s|$)/i', $class_name, $matches )
```

Comment:
```suggestion
			1 !== (int) preg_match( '/(?:^|\s)wp-image-([1-9]\d*)(?:\s|$)/i', $class_name, $matches )
```

Do we needs to cast `int` here?

Reply:
No, because it can either be an `int` or `false`, and we're checking to see if it is not identical to the value.